src/assets/networks-config.yaml

```diff
@@ -0,0 +1,32 @@
+# Paths to vde executables. Because vde_vmnet is invoked via sudo it should be
+# installed where only root can modify/replace it. This means also none of the
+# parent directories should be writable by the user.
+#
+# The varRun directory also must not be writable by the user because it will
+# include the vde_vmnet pid files. Those will be terminated via sudo, so replacing
+# the pid files would allow killing of arbitrary privileged processes. varRun
+# however MUST be writable by the daemon user.
+#
+# None of the paths segments may be symlinks, which is why it has to be /private/var
+# instead of /var etc.
+paths:
+  vdeSwitch: /opt/rancher-desktop/bin/vde_switch
+  vdeVMNet: /opt/rancher-desktop/bin/vde_vmnet
+  varRun: /private/var/run/rancher-desktop-lima
+  sudoers: /private/etc/sudoers.d/rancher-desktop-lima
+group: staff
+networks:
+  shared:
+    mode: shared
+    gateway: 192.168.205.1
+    dhcpEnd: 192.168.205.254
+    netmask: 255.255.255.0
+  bridged:
+    mode: bridged
+    interface: en0
```

Comment:
The user can edit this file (also to resolve a subnet conflict with a local private network or VPN).

We eventually should make this configurable via the UI, but for now this is what we have.

`en0` _should_ be the primary interface of the machine, but I guess we need to document how the user can change it.

Reply:
Oh, just realized: we don't actually use the `bridged` network by default, we use `shared`. So unless the user changes the `networks:` settings in `lima.yaml`, this will not make any difference anyways. And I'm not sure if RD won't revert any changes the user makes to `networks`, so this may not be configurable yet at all.